### DISCUSSION TITLE
Install hangs as soon as option selected

### DISCUSSION THREAD

**Discussion by terryrey** [Security-Onion-Solutions/securityonion#8358] [Unsupported Versions] [answered]
Hi,

First off I had better confess I am a complete and utter noob to SecurityOnion so, if I have done/not done something blatantly obvious, or missed something in the documentation, I apologise in advance.

With that out of the way I am trying to create a home setup to test & learn with. I'm using a new machine that will be dedicated to this.  The problem I am having is that I boot using the ISO and get as far as the installation options, but, as soon as I choose an option, I get a screen that says SecurityOnion and goes no further.  

I have read the documentation and see that a minimum memory requirement is 16GB.  I currently only have 8GB and am waiting on the second chip to come, sometime next week.  

My question is, should I still be able to install with only 8GB?  If so, then what can you please suggest I look at to complete the install?

I have checked that virtualization is enabled in Bios.

Thanks
Terry

**Comment by isaacgolding** [comment]
So there is a lot going on under the hood for SO and if you are picking standalone option then you are already under the required memory for the install.  This isn't a case where you can squeek by under the min specs... things really don't work without enough ram and the install script actually checks to make sure you have the min memory for the install option you picked.

Minimum Specs
If you just want to import a pcap using [so-import-pcap](https://docs.securityonion.net/en/2.3/so-import-pcap.html#so-import-pcap), then you can configure Security Onion 2 as an Import Node with the following minimum specs:

4GB RAM
2 CPU cores
200GB storage

For all other configurations, the minimum specs for running Security Onion 2 are:

**12GB RAM**
4 CPU cores
200GB storage

from: https://docs.securityonion.net/en/2.3/hardware.html

**Comment by terryrey** [comment]
Thank you for the response.

I should have said that I had the same result with all the install options, 

The specs of the machine are 
2 CPU cores
16GB RAM (eventually)
1TB SSD

I will wait until I've got the rest of the memory before trying again.

Thanks
Terry.

**Comment by terryrey** [comment, marked as answer]
I finally got there and post this as it may help someone else.

I thought I'd install CentOS as a last resort whilst I'm waiting for the extra memory and had some issues with that too as it booted the ISO with a grub interface.

I then remembered the SSD was taken from an old Win10 machine so I deleted the partitions, tried SO again and to my surprise, it worked.

Thanks
Terry.
